moztelemetry/dataset.py

```diff
@@ -231,7 +258,7 @@ def summaries(self, sc, limit=None):
         keys = sc.parallelize(scanned).flatMap(self.store.list_keys)
         return keys.take(limit) if limit else keys.collect()
 
-    def records(self, sc, limit=None, sample=1, seed=42, decode=None, summaries=None):
+    def records(self, sc, group_by, limit=None, sample=1, seed=42, decode=None, summaries=None):
```

Comment:
`group_by` should have a single default value: `group_by='greedy'`. Can you add the new parameter to the docstring?